examples/example-vite-kitchen-sink/src/dojo/provider.tsx

```
@@ -0,0 +1,45 @@
+import type { SDK } from "@dojoengine/sdk";
+import { env, getRpcUrl } from "@/env";
+
+export function useDojoDb() {
+  return useContext(DojoContext);
+}
+
+export const DojoContext = createContext<SDK<OnchainDashSchemaType> | null>(null);
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface DojoProviderProps {
+}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)